lib/ansible/modules/cloud/misc/proxmox.py

```diff
@@ -175,6 +175,14 @@
      - Indicate desired state of the instance
     choices: ['present', 'started', 'absent', 'stopped', 'restarted']
     default: present
+  pubkey:
+    version_added: "2.3"
+    description:
+      - Public key to add to /root/.ssh/authorized_keys
```

Comment:
`Public key to add to .ssh/authorized_keys.`
Missing full stop
May not be run as root